examples/example_mc_fit_a_spectrum.py

```diff
@@ -51,6 +55,7 @@
     mcfit = spectrum.mcfit(obs, samples=1024, multiprocessing=False)
 
     # Create plots.
-    mcfit.plot(wavelength=True, charge=True)
-    mcfit.plot(wavelength=True, size=True)
-    mcfit.plot(wavelength=True, composition=True)
+    mcfit.plot(wavelength=True, charge=True, ptype='charge', units=units)
```

Comment:
I'm a bit confused by having both charge=True and ptype='charge', and alike ...

Reply:
Yes I agree. The other way to do it is like:

            if keywords.get("charge"):
                ptype = "charge"
            elif keywords.get("size"):
                ptype = "size"
            elif keywords.get("composition"):
                ptype = "composition"
            elif keywords.get("residual"):
                ptype = "residual"
            else:
                ptype = "fitted"

and only keep charge/size/etc=Truth when calling plot. This would reduce the coverage even more I think with more if statements.

Reply:
This would work, or, you could select on keywords.get(x) instead of introducing the new variable ptype. Presumable, to increase coverage you would need to have a test for each case.

Reply:
The if statements would still be repeated and instead of `ptype = "charge/size/etc"` it would be `fig.savefig(f"{keywords['output']}/{keywords.get(charge/size/etc)}.{keywords['ftype']}")` and the else would have fitted there. I think visually is better to just introduce ptype and have fig.savefig once. Either way is fine, let me know which one you prefer.

Reply:
I agree, using ptype is clear code.